src/MultiMessageSender.sol

```diff
@@ -105,19 +106,28 @@ contract MultiMessageSender {
         /// @dev increments nonce
         ++nonce;
 
+        uint256 msgExpiration = block.timestamp + gac.getMsgExpiryTime();
+
         MessageLibrary.Message memory message =
-            MessageLibrary.Message(_dstChainId, _target, nonce, _callData, _expiration, "");
+            MessageLibrary.Message(block.chainid, _dstChainId, _target, nonce, _callData, msgExpiration, "");
 
         for (uint256 i; i < adapterLength;) {
             IBridgeSenderAdapter bridgeAdapter = IBridgeSenderAdapter(adapters[i]);
             message.bridgeName = bridgeAdapter.name();
```

Comment:
is there a good reason for having the `bridgeName` attribute as part of the Message payload? It is the one field that is a property of the communication channel rather than the message, and we treat it separately in a few places because of this. 
Similar to the `IBridgeSenderAdapter` interface, can we instead just have `IBridgeReceiverAdapter` provide a `name()` for the associated bridge? That way on the destination chain, we can infer the bridge name from the adapter that delivers the message, if it is required.

Reply:
I think this "bridge name in message" was introduced during initial development without thorough consideration. Now I agree that moving it to the receiver adapter is better.